src/OpenTelemetry.Exporter.Geneva/OpenTelemetry.Exporter.Geneva.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <Description>An OpenTelemetry .NET exporter that exports to local ETW or UDS</Description>
+    <Authors>OpenTelemetry Authors</Authors>
+    <NoWarn>$(NoWarn),NU5104,CS1591,SA1123,SA1633,SA1310,CA1031,CA1810,CA1822,CA2000,CA2208,SA1204,SA1201,SA1202,SA1308,SA1309,SA1311,SA1402,SA1602,SA1649</NoWarn>
```

Comment:
🙈

Reply:
Ya, the code style in the other repository was a bit different than this one. I figured ignoring most of the harder to fix warnings was the best way to go for now, and eventually work on converging the rules with the repo as a whole :)